GenumKit/Parsers/AssetsCatalogParser.swift

```diff
@@ -13,7 +13,7 @@ public final class AssetsCatalogParser {
   public init() {}
 
   @discardableResult
-  public func addImageName(_ name: String) -> Bool {
+    public func addImage(named name: String) -> Bool {
```

Comment:
Huh, why the indentation change here? Seems like a misindentation was introduced by mistake